courses/his201/quizz/008/fr.yml

```
@@ -1,4 +1,4 @@
-question: Qu'était e-gold ?
+question: Qu'était-ce e-gold ?
```

```suggestion
question: Qu'était e-gold ?
```

"Qu'était" is right here imho